std/algorithm/sorting.d

```diff
@@ -846,6 +846,47 @@ private template validPredicates(E, less...)
             && validPredicates!(E, less[1 .. $]);
 }
 
+private bool multiSortPredFun(Range, funs...)(ElementType!Range a, ElementType!Range b) 
+{
+    foreach (f; funs)
+    {
+        alias lessFun = binaryFun!(f);
+        if (lessFun(a, b)) return true;
+        if (lessFun(b, a)) return false;
+    }
+    return false;
+}
+
+private void multiSortImpl(Range, SwapStrategy ss, funs...)(Range r)
+{
```

Comment:
nitpick: the typical style is too put an private implementation below the public declaration, s.t. it's more obvious to a reader that might scroll top to bottom.